src/action/common/create_nix_tree.rs

```diff
@@ -8,6 +8,7 @@ use crate::action::{
 };
 
 const PATHS: &[&str] = &[
+    "/nix",
```

Comment:
I don't think we want to do it as part of `CreateNixTree`, as that happens before we hit the revert for `nix.mount`, etc in the ostree planner, which will probably cause issues.

We'll probably want to create something similar to `RevertCleanSteamosNixOffload` for the ostree planner that does nothing in `execute`, but deletes `/nix` in `revert`, and have it be one of the first steps in the plan (so it's one of the last steps to be reverted upon uninstall).